x/bank/internal/types/events.go

```diff
@@ -6,6 +6,7 @@ var (
 
 	AttributeKeyRecipient = "recipient"
 	AttributeKeySender    = "sender"
+	AttributeKeyAmount    = "amount"
```

Comment:
As this key is used also on staking and distribution, I think we can move it to the `/types` package